src/containers/container.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Menu from '../components/menu';
+import NoteList from '../components/note-list';
+import Note from '../components/note';
+
+export default class Container extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleOnClick = this.handleOnClick.bind(this);
+    this.state = {selectedNoteId: null};
+  }
+
+  handleOnClick(id) {
+    this.setState({selectedNoteId: id});
+  }
+
+  render() {
+    const selectedNoteId = this.state.selectedNoteId;
+    let noteToDisplay = {};
+    for (let note of this.props.data) {
```

Comment:
Using `.find()` in this case.
Eg: `data.find(note => node.id === selectedNoteId);`